packages/wrangler/src/assets.ts

```diff
@@ -138,7 +138,7 @@ export const syncAssets = async (
 						[(await readFile(absFilePath)).toString("base64")],
 						manifestEntry[1].hash,
 						{
-							type: getContentType(absFilePath),
+							type: getContentType(absFilePath) || "application/octet-stream",
```

Comment:
For future readers, this is the line that allows us to gradually roll this feature out before production support is in place.